debug.log

```diff
@@ -0,0 +1 @@
+[1031/022935.637:ERROR:exit_code_watcher_win.cc(87)] Failed to wait for process exit or stop event
```

Comment:
Just curious do you need log files in VCS? may be it you can consider to add *.log into .gitignore